test.c

```diff
@@ -0,0 +1,32 @@
+#include<stdio.h>
+
+// Define an enumeration
+enum Day {
+    Monday,
+    Tuesday,
+    Wednesday,
+    Thursday,
+    Friday,
+    Saturday,
+    Sunday
+};
+enum Month{
+    January,
+    February,
+    March,
+    April,
+    May,
+    June,
+    July,
+    August,
+    September,
+    October,
+    November,
+    December
+};
+int main() {
+    printf("Symbolic Name of %d : %s \n",Day::Thursday,__nameof(Day::Thursday));
+    printf("Symbolic Name of %d : %s \n",Month::February, __nameof(Month::February));
+    return 0;
```

Comment:
Add a LIT test case, not this. Checkout example LIT test cases existing in test/ directories.

Reply:
Why do you still have this test case? Only add LIT test cases.